Does not build on i586 (GTAV plugin fails)

### Description

When building for i586 the GTAV Plugin fails (which is ok as the game only works on amd64), but rather then failing the i586 build should default to disable amd64 only plugins.

### Steps to reproduce

build on i586

### Mumble version

1.4.274

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

Caused by memory alignment, which is different on 32 bit and 64 bit machines:

```
[ 1615s] In file included from /home/abuild/rpmbuild/BUILD/mumble-src/plugins/gtav/Game.h:9,
[ 1615s]                  from /home/abuild/rpmbuild/BUILD/mumble-src/plugins/gtav/gtav.cpp:6:
[ 1615s] /home/abuild/rpmbuild/BUILD/mumble-src/plugins/gtav/structs.h:218:37: error: static assertion failed
[ 1615s]   218 | static_assert(sizeof(CCameraAngles) == 0x408, "");
[ 1615s]       |               ~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~
[ 1615s] /home/abuild/rpmbuild/BUILD/mumble-src/plugins/gtav/structs.h:218:37: note: the comparison reduces to '(1028 == 1032)'
```

### Relevant log output

_No response_

### Screenshots

_No response_

--- a/plugins/gtav/structs.h
+++ b/plugins/gtav/structs.h
@@ -118,7 +118,7 @@ struct CCameraManagerAngles {
 	ptr_t cameraAngles; // CCameraAngles *
 };
 
-struct CCameraAngles {
+struct alignas(8) CCameraAngles {
 	uint8_t pad1[960];
 	ptr_t playerAngles; // CPlayerAngles *
 	uint8_t pad2[60];

